src/config/client-config.c

```diff
@@ -322,14 +298,7 @@ int Read_Client_Server(XML_NODE node, agent * logr)
             }
             network_interface = (uint32_t)interface_numeric;
         } else if (strcmp(node[j]->element, xml_protocol) == 0) {
-            if (strcmp(node[j]->content, "tcp") == 0) {
-                protocol = IPPROTO_TCP;
-            } else if (strcmp(node[j]->content, "udp") == 0) {
-                protocol = IPPROTO_UDP;
-            } else {
-                merror(XML_VALUEERR, node[j]->element, node[j]->content);
-                return (OS_INVALID);
-            }
+            mwarn("Client option 'protocol' is no longer configurable. Method TCP is the only one allowed.");
```

Comment:
Should we consider using [this format](https://github.com/wazuh/wazuh/pull/31372/files#diff-397a19fce00622d9b7d0efd77884f5146133a5bdd2c50a30f786f907625411b6R175) to update this message?

```suggestion
            mwarn("Deprecated option 'protocol' is no longer available. Method TCP is the only one allowed.");
```